frontend/amundsen_application/api/quality/v0.py

```diff
@@ -0,0 +1,59 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import logging
+from pkg_resources import iter_entry_points
+
+from http import HTTPStatus
+
+from flask import Response, jsonify, make_response, request, current_app as app
+from flask.blueprints import Blueprint
+from marshmallow import ValidationError
+from werkzeug.utils import import_string
+
+LOGGER = logging.getLogger(__name__)
+QUALITY_CLIENT_CLASS = None
+QUALITY_CLIENT_INSTANCE = None
+
+for entry_point in iter_entry_points(group='quality_client', name='quality_client_class'):
+    quality_client_class = entry_point.load()
+    if quality_client_class is not None:
+        QUALITY_CLIENT_CLASS = quality_client_class
+
+quality_blueprint = Blueprint('quality', __name__, url_prefix='/api/quality/v0')
```

Comment:
what's the guidance re configuring this client. Should it be done via an entry point or via app.config? Right now you try entry point and then check app.config later if entry point didn't work. 

I prefer sticking to just one way, and I lean towards app.config.

Reply:
That would also allow eliminating the globals, which is gr8

Reply:
I'm not really sure what's the preferred method. Most existing code (announcements, data preview,  supports both entry points and configs. I agree it's confusing and unnecessarily complicates things. I'll axe the entry point support here.